db/migrate/20210104164027_create_user_recipe.rb

```diff
@@ -5,6 +5,7 @@ def change
       t.references :recipe, null: false, foreign_key: true
       t.float :amount_donated
       t.float :recipe_rating
```

Comment:
One thought I had, was where will an individual rating be stored? I know this will be similar to what we've done on adopt don't shop/monster shop but I'm struggling to remember!

Reply:
My understanding is that the individual ratings would be stored here and whenever a user_recipe is created it would update the field avg_rating on recipes

Reply:
What Logan said. Basically my interpretation is when a new UserRecipe is created the rating is null and this does not yet affect that recipes average rating. However, once a user then updates their UserRecipe rating (ie. they tried it and found it to be amazing) then once the that UserRecipe rating is updated it will then trigger the update for the average rating on that Recipe.